### PR TITLE
Update .../docs/data-and-analytics/block-explorers

### PR DESCRIPTION
**Pull request by siddhantkharode**
Added the descriptions for block root, state root, graffiti, CommitteeIndex, Aggregation Bits, Beacon Block Root, source and target.
Added description of LUKSO L14 along with URL of the block explorer.
Removed a repetition on line 196 (Attestations).

### CODE REVIEW

File: src/content/developers/docs/data-and-analytics/block-explorers/index.md [hunk 1]
```diff
@@ -178,12 +178,12 @@ Slots are opportunities for block creation, the data available for each slot inc
...
 - Proposer – The validator that proposed the block for the slot
-- Block root – ?
+- Block root – The hash-tree-root of the BeaconBlock
 - Parent root – The hash of the block that came before
-- State root – ?
+- State root – The hash-tree-root of the BeaconState
 - Signature – ?
 - Randao reveal – ?
-- Graffiti
+- Graffiti – A block proposer can include 32 byte long message to its block proposal
```
Comment (wackerow) [inline src/content/developers/docs/data-and-analytics/block-explorers/index.md:186]: (small grammar suggestion)
- Graffiti – A block proposer can include a 32-byte long message to its block proposal

**Review by wackerow — CHANGES_REQUESTED** [review]
@siddhantkharode Thanks for filling this in! Looks much better now. Made one small grammar note, otherwise nice job 👍

File: src/content/developers/docs/data-and-analytics/block-explorers/index.md (hunk 1 above)
Comment (siddhantkharode) [reply src/content/developers/docs/data-and-analytics/block-explorers/index.md:186]: Thanks!

**Review by samajammin — APPROVED** [review]
Thanks for this!

**Comment by samajammin** [comment]
@all-contributors please add @siddhantkharode for content.

**Pull request merged**